hack/bin/integration-setup-federation.sh

```diff
@@ -49,7 +49,20 @@ export FEDERATION_DOMAIN_2="federation-test-helper.$FEDERATION_DOMAIN_BASE"
 
 echo "Installing charts..."
 
+set +e
 helmfile --environment "$HELMFILE_ENV" --file "${TOP_LEVEL}/hack/helmfile.yaml" sync --skip-deps --concurrency 0
+EXIT_CODE=$?
+
+if (( EXIT_CODE > 0)); then
+    echo "!! Helm install failed. Attempting to get some more information ..."
+
+    kubectl -n "$NAMESPACE_1" get events | grep -v "Normal "
+    kubectl -n "$NAMESPACE_2" get events | grep -v "Normal "
```

Comment:
Grepping out `Normal` might create red herrings because I guess we won't know if things got better. How much noise does this save?

Reply:
A lot of noise. It's 95% normal events. The events only show if helm install fails - and then these warnings/errors are likely indicative of a problem, even if sometimes unrelated to the failure, I suppose often there's something to it.

Reply:
Makes sense :shipit: